doc/src/assets/custom.sty

```diff
@@ -0,0 +1,45 @@
+%% Load the macro package required for the cover.
+
+
+%% pkg for make cover page BEGIN ----------------------------------------------
+% Load `geometry' to modify margins later
+\usepackage{geometry}
+% "some": use \BgThispage to change background
+% ref: background@v2.1,# 2.1 Options, "pages="
+%   http://mirrors.ctan.org/macros/latex/contrib/background/background.pdf
+\usepackage[pages=some]{background}
+
+%% Color definitions for Julia
+%%  https://github.com/JuliaLang/julia-logo-graphics#color-definitions
+\definecolor{julia_blue}  {HTML}{4063D8}
+\definecolor{julia_green} {HTML}{389826}
+\definecolor{julia_purple}{HTML}{9558B2}
+\definecolor{julia_red}   {HTML}{CB3C33}
+\definecolor{splash_gary} {HTML}{1A1A33}
+
+% ---- define heading background
+% ref: background.pdf, #2.1 Options
+\backgroundsetup{
+scale=1,    % scaling factor
+angle=0,    % counterclockwise angle
+opacity=1,  % transparency
+contents={
+%% Place the background image `title-bg' in the right place via `tikz'.
+% tikz option "remember picture", "overlay"
+% ref: pgfmanual@3.1.9a, #17.13.1 Referencing a Node in a Different Picture\
+%   http://mirrors.ctan.org/graphics/pgf/base/doc/pgfmanual.pdf
+\begin{tikzpicture}[remember picture,overlay,draw=white]
+  \draw [path picture={
+    % ref: pgfmanual, 15.6, "Predefined node path picture bounding box"
+    \node at (path picture bounding box.center){
+      \input{assets/cover-splash}
+  };}] (-0.5\paperwidth,4cm) rectangle (0.5\paperwidth,11cm);
+  % Put picture to right place
+  %   ref: pgfmanual, #2.6 Rectangle Path Construction
+\end{tikzpicture}
+}}%
+
+% ---- Heading font style
+\DeclareFixedFont{\MainHeading}{T1}{phv}{b}{n}{1.5cm}
+\DeclareFixedFont{\SecondaryHeading}{T1}{phv}{b}{n}{0.8cm}
```

Comment:
@inkydragon I was able to [extract the logs from the Docker build](https://gist.github.com/mortenpi/36cb8d5cea8a3b56e1a7ea329e4da377). It looks like these fonts are missing:

```
! Font \T1/phv/b/n/42.67912=phvb8t at 42.6791pt not loadable: metric data not f
ound or bad.
<to be read again> 
relax 
l.43 ...ixedFont{\MainHeading}{T1}{phv}{b}{n}{1.5cm}
                                                  

luaotfload | resolve : sequence of 3 lookups yielded nothing appropriate.
! Font \T1/phv/b/n/22.76228=phvb8t at 22.7623pt not loadable: metric data not f
ound or bad.
<to be read again> 
relax 
l.44 ...ont{\SecondaryHeading}{T1}{phv}{b}{n}{0.8cm}
```

Would this be a fatal error?

Reply:
@mortenpi I'm not sure, but I had no issues compiling locally with `lualatex`

Can you reproduce the build failure? 
Or does the problem only happen with CI

Reply:
Yea, it's consistent if you build with the Docker image. To reproduce, if you have Docker installed, you can just run

```
DOCUMENTER_LATEX_DEBUG=$PWD/pdf-logs make -C doc pdf texplatform=docker
```

This will also copy the logs to `pdf-logs/`, although you need to update Documenter to the latest master to get useful LaTeX logs out.